todo_app/data/trello_items.py

```diff
@@ -0,0 +1,181 @@
+# Trello card actions
+#
+import requests
+import json
+from todo_app.data.Item_class import Item
+
+_DEFAULT_ITEMS = [
+    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
+    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
+]
```

Comment:
I think you never use this any more, if that's true we should delete it (and similarly, deleting the entire `session_items.py` file would be nice!)